fix(S3): content review and update (#2447)
scaleway/docs-content: storage/object/how-to/monitor-consumption.mdx

## How to activate your Cockpit

> Scaleway Cockpit is deactivated by default, but you can [follow our documentation](/observability/cockpit/how-to/activate-cockpit/) to activate with a few steps.

Reviewer: ```suggestion
Scaleway Cockpit is deactivated by default, but you can [read our documentation](/observability/cockpit/how-to/activate-cockpit/) to activate it in a few steps.
```

## How to access the Grafana dashboard

To view your Object Storage metrics and logs, use the Grafana dashboard, which is accessible from the Scaleway console:
  - First you must create a user and their associated credentials (a username and password) for Grafana.
  - Then use these credentials to access the Grafana dashboard for all your Scaleway products via the Scaleway console.

> 1. Create a Grafana user and [retrieve their username and password](/observability/cockpit/how-to/retrieve-grafana-credentials/).

Reviewer: ```suggestion
1. Create a Grafana user and [retrieve your username and password](/observability/cockpit/how-to/retrieve-grafana-credentials/).
```
